game/src/client/scenes/raceGame/RaceScene.ts

```diff
@@ -25,6 +25,11 @@ import { BackgroundSceneData } from "./BackgroundScene";
 import { QuestionSceneData } from "./QuestionScene";
 import { EventNames, sceneEvents, subscribeToEvent } from "./RaceGameEvents";
 
+/**
+ * This class will render the game board and everything no related to it. That also includes characters, tiles and items on the board.
```

Comment:
`This class will render the game board and **everything no related** to it.`
Le "no" sème la confusion dans cette phrase!

Reply:
Ah oui oops, je voulais dire "not"